db/external_sst_file_ingestion_job.cc

```diff
@@ -937,26 +937,6 @@ Status ExternalSstFileIngestionJob::AssignLevelAndSeqnoForIngestedFile(
         overlap_with_db = true;
         break;
       }
-
-      if (compaction_style == kCompactionStyleUniversal && lvl != 0) {
```

Comment:
It seems sequence number is not assigned if overlap_with_db is false?

Reply:
Under overlap_with_db == false, we will have *assigned_seqno = 0 from 4192a0cc2c0b6320e1b27f28c28e9468765474ebc94097446b82380263854R888